src/query-builder.ts

```diff
@@ -36,8 +36,8 @@ export function fql(
  * function
  */
 export class Query {
-  readonly #queryFragments: ReadonlyArray<string>;
-  readonly #queryArgs: (QueryValue | Query)[];
+  readonly queryFragments: ReadonlyArray<string>;
+  readonly queryArgs: (QueryValue | Query)[];
```

Comment:
By moving encoding to strictly the TaggedTyped file we've now made these previously private fields public.

That feels like a regression as we don't want users to take any dependencies on these.

IMO it would be better to keep these private even it means we don't have as clean a break between files as we'd like.

Are there reasons we don't want to do some encoding here other than modular purity?